pkgs/development/libraries/gettext/default.nix

```diff
@@ -15,6 +15,7 @@ stdenv.mkDerivation rec {
       url = "https://git.savannah.gnu.org/gitweb/?p=gettext.git;a=patch;h=dce3a16e5e9368245735e29bf498dcd5e3e474a4";
       sha256 = "1lpjwwcjr1sb879faj0xyzw02kma0ivab6xwn3qciy13qy6fq5xn";
     })
+    ./CVE-2018-18751-bison.patch
```

Comment:
Could you hide this under `lib.optional stdenv.isDarwin` so that this is no rebuild on linux?

Reply:
Done in 9bc0d0efb7c11027139de24c1c415e694757bc8b.